cmd/config/cluster-maximums/cluster-maximums.yml

```diff
@@ -0,0 +1,141 @@
+# Meant to be run on a 24 node cluster with max-pods per node configured to 2000
+# Fills with pods the worker nodes. Creates 1 routes and service per 4 pods
+global:
+  gc: {{.GC}}
+  indexerConfig:
+    esServers: ["{{.ES_SERVER}}"]
+    insecureSkipVerify: true
+    defaultIndex: {{.ES_INDEX}}
+    type: {{.INDEXING_TYPE}}
+jobs:
+  - name: pods-per-node
+    namespace: pods-per-node
+    jobIterations: {{.JOB_ITERATIONS}}
+    qps: {{.QPS}}
+    burst: {{.BURST}}
+    namespacedIterations: true
+    iterationsPerNamespace: 50
+    waitWhenFinished: true
+    preLoadImages: true
+    preLoadPeriod: 15s
+    jobPause: 2m
+    namespaceLabels:
+      security.openshift.io/scc.podSecurityLabelSync: false
+      pod-security.kubernetes.io/enforce: privileged
+      pod-security.kubernetes.io/audit: privileged
+      pod-security.kubernetes.io/warn: privileged
+    objects:
+      - objectTemplate: deployment-server.yml   # TODO: Consider using different client/server applications. Use apache benchmark
+        replicas: 1
+        inputVars:
+          podReplicas: 2
+      - objectTemplate: deployment-client.yml
+        replicas: 1
+        inputVars:
+            podReplicas: 2
+            ingressDomain: {{.INGRESS_DOMAIN}}
+      - objectTemplate: service.yml
+        replicas: 1
+      - objectTemplate: route.yml
```

Comment:
Why are we creating a route for max pods per node test?

Reply:
I wanted to add some extra load to the data plane, not only stupid pods doing nothing

Reply:
Gotcha. I am not crazy about having the ingress overhead, but on the other hand, it would be a good test for the route->pod linkage for large numbers of pods.